.circleci/workflows.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       - run: npm install yarn --location=global
       - run: npm install solhint --location=global
       - run: npm install ganache --location=global
-      - run: npm install prettier prettier-plugin-solidity@1.0.0-beta.19 --location=global
```

Comment:
I wonder whether we should also do the same thing for yarn, solhint and ganache.
Then all the packages would be installed locally, under `node_modules`, right?

Reply:
We could do that yes. 
The only difference, when having them locally installed, would be in the way they are being invoked.

For globally installed npm packages, most of the times they are being aliased therefore you can run them as any other OS installed package (for example by running the command `ganache`).

For locally installed npm packages, they are usually ran by using `npx` which is a package runner. `npx` comes as a part of `npm` so no need for extra commands/dependencies.

I have to see where they are being used, and how they are used and adjust that accordingly.